airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -8,22 +8,18 @@ tests:
     - config_path: "secrets/config.json"
       status: "succeed"
     - config_path: "integration_tests/invalid_config.json"
-      status: "excefailedption"
+      status: "failed"
   discovery:
     - config_path: "secrets/config.json"
   basic_read:
     - config_path: "secrets/config.json"
-      configured_catalog_path: "sample_files/configured_catalog.json"
-# TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
-#      expect_records:
-#        path: "integration_tests/expected_records.txt"
-#        extra_fields: no
-#        exact_order: no
-#        extra_records: yes
-#  incremental: # TODO if your connector does not implement incremental sync, remove this block
-#    - config_path: "secrets/config.json"
-#      configured_catalog_path: "integration_tests/configured_catalog.json"
-#      state_path: "integration_tests/abnormal_state.json"
+      configured_catalog_path: "integration_tests/product_catalog_v2_configured_catalog.json"
+      empty_streams:
```

Comment:
can you add a comment explaining why this is empty? `v1` etc

Reply:
I’m unable to create orders using API and I didn’t find a way to create orders through UI, so that’s why `orders` stream is empty.

<img width="1429" alt="image" src="https://user-images.githubusercontent.com/19872253/127700003-9aca1b83-1036-4f02-8ea4-286215e2296d.png">

Reply:
Done.